### PR TITLE
PR asks to notes licences on the wiki

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This would ensure that once #11825 is done, we are not forgetting any new dependencies.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -28,3 +28,4 @@ _Please, go through these checks before submitting the PR._
 - [ ] You have performed a self-review of your own code
 - [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
 - [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)
+- [ ] I added all extra licences, dependencies, libraries to [the wiki](https://github.com/ankidroid/Anki-Android/wiki/Licences)
```
Comment (lukstbit) [inline .github/pull_request_template.md:31]: How about:

_[ ] I recorded all changes to licences, dependencies, libraries to [the wiki](https://github.com/ankidroid/Anki-Android/wiki/Licences)_

?

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Comment by david-allison** [comment]
The more barriers we put up to entry, the less contributions we get.

Can we move this to a CI check?

**Comment by mikehardy** [comment]
> * Can we move the dependencies to a file with specific usage instructions?

Yes! #11101 - If there are no serious complaints, how bout we add this as a requirement for the final solution on that issue? It is meant to centralize dependency version management in one file and as part of that we can add a huge comment there about license attribution / credit ?

**Comment by Arthur-Milchior** [comment]
If all dependencies are in a single file, I see how you'd get the CI for dependencies.
However, for code that are ported to Kotlin from another language, which occurred at least twice (CSV and template parsing), I don't. 
But I'm also fine just relying on the fact that, once we have actually a list of licence, we'll trust reviewer to let contributors that translated code/imported it, to know one extra change is required,

**Comment by mikehardy** [comment]
I think we could actually change the template a little and have a **reviewer** checklist

That is, still have the reminder in there but in a second "Hey Reviewers, thanks for helping! Please check these things as you review:" list, and have a "Is there a new dependency added in this PR? Is the license for the dependency on this /link here/ page?"

Still a "light process" - as opposed to a CI check - but puts the barrier (if any) on a reviewer that can do it easily

**Comment by Arthur-Milchior** [comment]
I love your way of thinking Mike.

**Pull request closed without merging**